### DISCUSSION TITLE
Не вижу в кошельке токенов

### DISCUSSION THREAD

**Discussion by Micter7979** [Chia-Network/chia-blockchain#12714] [Support]
Hello! Please help me figure it out! 3.07.22 I transferred XCH tokens to Pawket wallet. Today I go in, the balance is 0. I started looking. There is no support service. On one of the sites I found a search for the address, entered the address that I indicated when withdrawing from the OKX Exchange, it really shows that my tokens are at this address, but not in the wallet. Can you please tell me what to do? It was advised to integrate the Pawket wallet into the Chia Blockhain wallet - N 44784893 - integrated no coins ((( Help, please return the tokens!

Hello! Please help me figure it out! 3.07.22 I transferred XCH tokens to Pawket wallet. Today I go in, the balance is 0. I started looking. There is no support service. On one of the sites I found a search for the address, entered the address that I indicated when withdrawing from the OKX Exchange, it really shows that my tokens are at this address, but not in the wallet. Can you please tell me what to do? It was advised to integrate the Pawket wallet into the Chia Blockhain wallet - N 44784893 - integrated no coins ((( Help, please return the tokens!

**Comment by androiddevkotlin** [comment]
The same bug. Chia is a shit coin. This guys are scammers
